internal/commands/validate/validate.go

```diff
@@ -104,6 +109,8 @@ func NewValidateCmd() *cobra.Command {
 		"set (-e A=B) or pass-through (-e A) an environment variable to the Starlark interpreter")
 	cmd.PersistentFlags().StringVarP(&validateFile, "file", "f", "",
 		"use file as the configuration file (the path should end with either .yml or ..star)")
+	cmd.PersistentFlags().BoolVarP(&shouldPrint, "print", "p", false,
+		"print the configuration as YAML (useful for debugging Starlak files)")
```

Comment:
It's called Starlark:

```suggestion
		"print the configuration as YAML (useful for debugging Starlark files)")
```

Note that there are also other instances of the same typo in this changeset.